File: example/src/main/java/com/amqtech/opensource/appintroexample/ui/fragment/MainTabsFragment.java
package com.amqtech.opensource.appintroexample.ui.fragment;

import android.graphics.Color;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.viewpager.widget.ViewPager;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.amqtech.opensource.appintroexample.ui.mainTabs.MainPagerAdapter;
import com.amqtech.opensource.appintroexample.util.TabLayout;
import com.github.paolorotolo.appintroexample.R;

/**
 * Created by andrew on 11/17/16.
 */

public class MainTabsFragment extends Fragment {

    ViewPager pager;
    MainPagerAdapter adapter;
    TabLayout tabs;
    CharSequence Titles[] = {"Layout 1", "Layout 2", "Custom Layout", "Custom Background"};
    int Numboftabs = 4;

    public MainTabsFragment() {
        //required empty constructor
    }

    @SuppressWarnings({"ConstantConditions", "deprecation"})
    @Override
    public void onActivityCreated(Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);

        adapter = new MainPagerAdapter(getChildFragmentManager(), Titles, Numboftabs);

        pager = getView().findViewById(R.id.mainPager);
        pager.setAdapter(adapter);

        tabs = getView().findViewById(R.id.mainTabs);
        tabs.setBackgroundColor(Color.parseColor("#1976D2"));
        tabs.setDistributeEvenly(false);
        tabs.setCustomTabColorizer(new TabLayout.TabColorizer() {
            @Override
            public int getIndicatorColor(int position) {
                return Color.parseColor("#1976D2");
            }
        });
        tabs.setViewPager(pager);
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_main_tabs, container, false);
    }
}

File: example/src/main/java/com/github/appintro/example/ui/fragment/MainTabsFragment.kt
package com.github.appintro.example.ui.fragment

import android.graphics.Color
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.github.appintro.appintroexample.R
import com.github.appintro.example.ui.mainTabs.MainPagerAdapter
import kotlinx.android.synthetic.main.fragment_main_tabs.*

class MainTabsFragment : Fragment() {
    private val titles = arrayOf(
            "Layout 1",
            "Layout 2",
            "Custom Layout",
            "Custom Background")

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)

        mainPager.adapter = MainPagerAdapter(childFragmentManager, titles)

        with(mainTabs) {
            setBackgroundColor(Color.parseColor("#1976D2"))
            setDistributeEvenly(false)
            setCustomTabColorizer { Color.parseColor("#1976D2") }
            setViewPager(mainPager)
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? =
            inflater.inflate(R.layout.fragment_main_tabs, container, false)
}